os/kernel/Kconfig

```diff
@@ -1013,7 +1014,7 @@ config LOG_DUMP_MAX_SIZE
 	default 262144
 
 config LOG_DUMP_MIN_SIZE
```

Comment:
We change the policy of minimum size of log dump.
Atleast 16KB is used regardless of free memory size → keep just 1 node when free memory is less than 256KB to guarantee application operation
Is this config still necessary? Could you check?

Reply:
yes, changes to support 1 node size if free memory is less than 256KB